src/index.js

```diff
@@ -151,7 +151,7 @@ module.exports = function (config = {}) {
 			log.all(id, 'looking in aliases')
 			for (let [aliasName, aliasPath] of Object.entries(alias)) {
 				log.all(id, 'looking in aliases', aliasName)
-				if (id.startsWith(aliasName + path.sep)) {
+				if (id.startsWith(aliasName + '/' || aliasName + '\\')) {
```

Comment:
maybe you meant `if (id.startsWith(aliasName + '/') || id.startsWith(aliasName + '\\')) {`

Reply:
Sorry yes you're right!